src/resources/views/master-product-variant/_form.blade.php

```diff
@@ -22,6 +22,22 @@
 
 <hr>
 
+<div class="mb-3 row">
+    <label class="col-form-label col-form-label-sm col-md-2">{{ __('State') }}</label>
+    <div class="col-md-4">
+        {{ Form::select('state', $states, null, [
+                'class' => 'form-select form-select-sm' . ($errors->has('state') ? ' is-invalid': ''),
```

Comment:
Add null via the placeholder `--`, see: https://github.com/vanilophp/admin/blob/4.1.1/src/resources/views/channel/_form.blade.php#L55